2023-12-13 01:20:29,819 - DEBUG - 
def chunks(task, it, n):
    return 1

2023-12-13 01:20:29,820 - DEBUG - 
def unlock_chord(self, group_id, callback, interval=0, max_retries=1, result=2, Result=3, GroupResult=4, result_from_tuple=5, **kwargs):
    return 1

2023-12-13 01:20:29,820 - DEBUG - 
def perform_get_position(user_id, task_id):
    return 1

2023-12-13 01:20:29,820 - DEBUG - 
def group(self, tasks, result, group_id, partial_args, add_to_parent=0):
    return 1

2023-12-13 01:20:29,820 - DEBUG - 
def xmap(task, it):
    return 1

2023-12-13 01:20:29,821 - DEBUG - 
def chain(*args, **kwargs):
    return 1

2023-12-13 01:20:29,821 - DEBUG - 
def backend_cleanup():
    return 1

2023-12-13 01:20:29,821 - DEBUG - 
def xstarmap(task, it):
    return 1

2023-12-13 01:20:29,821 - DEBUG - 
def chord(self, header, body, partial_args=0, interval=1, countdown=2, max_retries=3, eager=4, **kwargs):
    return 1

2023-12-13 01:20:29,822 - DEBUG - 
def accumulate(self, *args, **kwargs):
    return 1

2023-12-13 01:20:29,841 - DEBUG - Setting default socket timeout to 30
2023-12-13 01:20:29,841 - INFO - beat: Starting...
2023-12-13 01:20:29,875 - DEBUG - Current schedule:
<ScheduleEntry: get_position get_position() <freq: 30.00 seconds>
2023-12-13 01:20:29,875 - DEBUG - beat: Ticking with max interval->5.00 minutes
2023-12-13 01:20:29,876 - DEBUG - beat: Waking up in 29.98 seconds.
2023-12-13 01:20:59,856 - DEBUG - beat: Synchronizing schedule...
2023-12-13 01:20:59,873 - INFO - Scheduler: Sending due task get_position (get_position)
2023-12-13 01:20:59,895 - ERROR - Message Error: Couldn't apply scheduled task get_position: perform_get_position() missing 2 required positional arguments: 'user_id' and 'task_id'
['  File "/Users/lichaoyuan/Desktop/copytrade/venv/bin/celery", line 8, in <module>\n    sys.exit(main())\n', '  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/celery/__main__.py", line 15, in main\n    sys.exit(_main())\n', '  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/celery/bin/celery.py", line 236, in main\n    return celery(auto_envvar_prefix="CELERY")\n', '  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/click/core.py", line 1157, in __call__\n    return self.main(*args, **kwargs)\n', '  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/click/core.py", line 1078, in main\n    rv = self.invoke(ctx)\n', '  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/click/core.py", line 1688, in invoke\n    return _process_result(sub_ctx.command.invoke(sub_ctx))\n', '  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/click/core.py", line 1434, in invoke\n    return ctx.invoke(self.callback, **ctx.params)\n', '  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/click/core.py", line 783, in invoke\n    return __callback(*args, **kwargs)\n', '  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/click/decorators.py", line 33, in new_func\n    return f(get_current_context(), *args, **kwargs)\n', '  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/celery/bin/base.py", line 134, in caller\n    return f(ctx, *args, **kwargs)\n', '  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/celery/bin/beat.py", line 72, in beat\n    return beat().run()\n', '  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/celery/apps/beat.py", line 84, in run\n    self.start_scheduler()\n', '  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/celery/apps/beat.py", line 113, in start_scheduler\n    service.start()\n', '  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/celery/beat.py", line 643, in start\n    interval = self.scheduler.tick()\n', '  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/celery/beat.py", line 353, in tick\n    self.apply_entry(entry, producer=self.producer)\n', '  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/celery/beat.py", line 283, in apply_entry\n    exc, traceback.format_stack(), exc_info=True)\n']
Traceback (most recent call last):
  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/celery/beat.py", line 402, in apply_async
    return task.apply_async(entry_args, entry_kwargs,
  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/celery/app/task.py", line 559, in apply_async
    check_arguments(*(args or ()), **(kwargs or {}))
TypeError: perform_get_position() missing 2 required positional arguments: 'user_id' and 'task_id'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/celery/beat.py", line 280, in apply_entry
    result = self.apply_async(entry, producer=producer, advance=False)
  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/celery/beat.py", line 410, in apply_async
    reraise(SchedulingError, SchedulingError(
  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/celery/exceptions.py", line 109, in reraise
    raise value.with_traceback(tb)
  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/celery/beat.py", line 402, in apply_async
    return task.apply_async(entry_args, entry_kwargs,
  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/celery/app/task.py", line 559, in apply_async
    check_arguments(*(args or ()), **(kwargs or {}))
celery.beat.SchedulingError: Couldn't apply scheduled task get_position: perform_get_position() missing 2 required positional arguments: 'user_id' and 'task_id'
2023-12-13 01:20:59,898 - DEBUG - beat: Waking up in 29.96 seconds.
2023-12-13 01:42:30,340 - INFO - beat: Starting...
2023-12-13 01:42:30,362 - INFO - Scheduler: Sending due task get_position (get_position)
2023-12-13 01:42:30,364 - ERROR - Message Error: Couldn't apply scheduled task get_position: perform_get_position() missing 2 required positional arguments: 'user_id' and 'task_id'
['  File "/Users/lichaoyuan/Desktop/copytrade/venv/bin/celery", line 8, in <module>\n    sys.exit(main())\n', '  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/celery/__main__.py", line 15, in main\n    sys.exit(_main())\n', '  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/celery/bin/celery.py", line 236, in main\n    return celery(auto_envvar_prefix="CELERY")\n', '  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/click/core.py", line 1157, in __call__\n    return self.main(*args, **kwargs)\n', '  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/click/core.py", line 1078, in main\n    rv = self.invoke(ctx)\n', '  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/click/core.py", line 1688, in invoke\n    return _process_result(sub_ctx.command.invoke(sub_ctx))\n', '  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/click/core.py", line 1434, in invoke\n    return ctx.invoke(self.callback, **ctx.params)\n', '  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/click/core.py", line 783, in invoke\n    return __callback(*args, **kwargs)\n', '  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/click/decorators.py", line 33, in new_func\n    return f(get_current_context(), *args, **kwargs)\n', '  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/celery/bin/base.py", line 134, in caller\n    return f(ctx, *args, **kwargs)\n', '  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/celery/bin/beat.py", line 72, in beat\n    return beat().run()\n', '  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/celery/apps/beat.py", line 84, in run\n    self.start_scheduler()\n', '  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/celery/apps/beat.py", line 113, in start_scheduler\n    service.start()\n', '  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/celery/beat.py", line 643, in start\n    interval = self.scheduler.tick()\n', '  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/celery/beat.py", line 353, in tick\n    self.apply_entry(entry, producer=self.producer)\n', '  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/celery/beat.py", line 283, in apply_entry\n    exc, traceback.format_stack(), exc_info=True)\n']
Traceback (most recent call last):
  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/celery/beat.py", line 402, in apply_async
    return task.apply_async(entry_args, entry_kwargs,
  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/celery/app/task.py", line 559, in apply_async
    check_arguments(*(args or ()), **(kwargs or {}))
TypeError: perform_get_position() missing 2 required positional arguments: 'user_id' and 'task_id'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/celery/beat.py", line 280, in apply_entry
    result = self.apply_async(entry, producer=producer, advance=False)
  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/celery/beat.py", line 410, in apply_async
    reraise(SchedulingError, SchedulingError(
  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/celery/exceptions.py", line 109, in reraise
    raise value.with_traceback(tb)
  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/celery/beat.py", line 402, in apply_async
    return task.apply_async(entry_args, entry_kwargs,
  File "/Users/lichaoyuan/Desktop/copytrade/venv/lib/python3.9/site-packages/celery/app/task.py", line 559, in apply_async
    check_arguments(*(args or ()), **(kwargs or {}))
celery.beat.SchedulingError: Couldn't apply scheduled task get_position: perform_get_position() missing 2 required positional arguments: 'user_id' and 'task_id'
2023-12-13 01:44:10,355 - INFO - beat: Starting...
2023-12-13 01:44:10,375 - INFO - Scheduler: Sending due task get_position (get_position)
2023-12-13 01:45:10,374 - INFO - Scheduler: Sending due task get_position (get_position)
2023-12-13 01:46:10,374 - INFO - Scheduler: Sending due task get_position (get_position)
2023-12-13 01:47:10,374 - INFO - Scheduler: Sending due task get_position (get_position)
2023-12-13 01:48:10,374 - INFO - Scheduler: Sending due task get_position (get_position)
2023-12-13 01:49:10,374 - INFO - Scheduler: Sending due task get_position (get_position)
2023-12-13 01:50:10,374 - INFO - Scheduler: Sending due task get_position (get_position)
2023-12-13 01:52:09,474 - INFO - beat: Starting...
2023-12-13 01:52:09,496 - INFO - Scheduler: Sending due task get_position (get_position)
2023-12-13 01:53:09,494 - INFO - Scheduler: Sending due task get_position (get_position)
2023-12-13 01:57:56,744 - INFO - beat: Starting...
2023-12-13 01:57:56,766 - INFO - Scheduler: Sending due task get_position (get_position)
2023-12-13 01:58:56,764 - INFO - Scheduler: Sending due task get_position (get_position)
2023-12-13 01:59:56,764 - INFO - Scheduler: Sending due task get_position (get_position)
2023-12-13 02:01:40,312 - INFO - beat: Starting...
2023-12-13 02:01:40,335 - INFO - Scheduler: Sending due task get_position (get_position)
2023-12-13 02:01:50,333 - INFO - Scheduler: Sending due task get_position (get_position)
2023-12-13 02:02:00,333 - INFO - Scheduler: Sending due task get_position (get_position)
2023-12-13 02:02:10,333 - INFO - Scheduler: Sending due task get_position (get_position)
2023-12-13 02:05:20,230 - INFO - Connected to redis://127.0.0.1:6379/14
2023-12-13 02:05:20,234 - INFO - mingle: searching for neighbors
2023-12-13 02:05:21,245 - INFO - mingle: all alone
2023-12-13 02:05:21,266 - INFO - celery@mac ready.
2023-12-13 02:05:21,282 - INFO - Task get_position[6a32b008-93e1-4467-8fbd-339d7068f5d5] received
2023-12-13 02:05:21,286 - INFO - Task get_position[b57a5773-8337-41fc-9025-8d7970985ad3] received
2023-12-13 02:05:21,290 - INFO - Task get_position[1e08f289-82bc-43e9-aebf-84859637f6bd] received
2023-12-13 02:05:21,294 - INFO - Task get_position[1a6edcbf-ae33-4c78-bcab-53e7107c5302] received
2023-12-13 02:05:21,299 - INFO - Task get_position[106398d9-77a9-44f9-9e48-76ababbd2037] received
2023-12-13 02:05:21,302 - INFO - Task get_position[d498cd47-9f7f-4653-9941-56fb0d74ba6e] received
2023-12-13 02:05:21,306 - INFO - Task get_position[6ff98c92-e530-45bc-8e94-86558221c35b] received
2023-12-13 02:05:21,308 - INFO - Task get_position[527c27b9-f5cf-4fb8-a3cc-e28250731c79] received
2023-12-13 02:05:21,312 - INFO - Task get_position[2f397b37-0a2e-4be2-9d28-a8cd1bffb037] received
2023-12-13 02:05:21,316 - INFO - Task get_position[75b4a49f-1027-4abb-a308-0e620a75b55e] received
2023-12-13 02:05:21,320 - INFO - Task get_position[4fd7131e-b6da-4735-a5f5-5fcf123e828f] received
2023-12-13 02:05:21,323 - INFO - Task get_position[809a0656-f180-4ddd-bc47-b42f273db66b] received
2023-12-13 02:05:21,326 - INFO - Task get_position[612bec87-b058-46fa-892e-82c1f88424e4] received
2023-12-13 02:05:21,329 - INFO - Task get_position[fd8670c4-f597-4694-99a7-de116abfc564] received
2023-12-13 02:05:21,334 - INFO - Task get_position[9783746d-8655-4c77-b041-a2cf93cf7f1c] received
2023-12-13 02:05:21,336 - INFO - Task get_position[5f2b7671-065d-4a6d-96ac-d92c843bddd0] received
2023-12-13 02:05:25,779 - INFO - Task get_position[b57a5773-8337-41fc-9025-8d7970985ad3] succeeded in 4.492487139000001s: None
2023-12-13 02:05:25,853 - INFO - Task get_position[1e08f289-82bc-43e9-aebf-84859637f6bd] succeeded in 4.554109924999999s: None
2023-12-13 02:05:26,237 - INFO - Task get_position[6a32b008-93e1-4467-8fbd-339d7068f5d5] succeeded in 4.948746060000001s: None
2023-12-13 02:05:26,251 - INFO - Task get_position[527c27b9-f5cf-4fb8-a3cc-e28250731c79] succeeded in 4.939084643000001s: None
2023-12-13 02:05:26,466 - INFO - Task get_position[1a6edcbf-ae33-4c78-bcab-53e7107c5302] succeeded in 5.169013656999997s: None
2023-12-13 02:05:26,539 - INFO - Task get_position[106398d9-77a9-44f9-9e48-76ababbd2037] succeeded in 5.233215969s: None
2023-12-13 02:05:26,593 - INFO - Task get_position[5f2b7671-065d-4a6d-96ac-d92c843bddd0] succeeded in 5.251711018999998s: None
2023-12-13 02:05:26,593 - INFO - Task get_position[75b4a49f-1027-4abb-a308-0e620a75b55e] succeeded in 5.274437966000001s: None
2023-12-13 02:05:26,594 - INFO - Task get_position[6ff98c92-e530-45bc-8e94-86558221c35b] succeeded in 5.282188521000002s: None
2023-12-13 02:05:26,617 - INFO - Task get_position[fd8670c4-f597-4694-99a7-de116abfc564] succeeded in 5.283362025999999s: None
2023-12-13 02:05:26,617 - INFO - Task get_position[612bec87-b058-46fa-892e-82c1f88424e4] succeeded in 5.2827116929999995s: None
2023-12-13 02:05:26,762 - INFO - Task get_position[9783746d-8655-4c77-b041-a2cf93cf7f1c] succeeded in 5.421501564s: None
2023-12-13 02:05:26,867 - INFO - Task get_position[d498cd47-9f7f-4653-9941-56fb0d74ba6e] succeeded in 5.561513422000001s: None
2023-12-13 02:05:26,869 - INFO - Task get_position[809a0656-f180-4ddd-bc47-b42f273db66b] succeeded in 5.5428258889999995s: None
2023-12-13 02:05:26,870 - INFO - Task get_position[4fd7131e-b6da-4735-a5f5-5fcf123e828f] succeeded in 5.542832541999999s: None
2023-12-13 02:05:26,945 - INFO - Task get_position[2f397b37-0a2e-4be2-9d28-a8cd1bffb037] succeeded in 5.625280987s: None
2023-12-13 02:05:49,410 - INFO - Task get_position[e9e3b0c1-f4ae-4440-9587-8cdbbca707f3] received
2023-12-13 02:05:53,675 - INFO - Task get_position[e9e3b0c1-f4ae-4440-9587-8cdbbca707f3] succeeded in 4.263323999000001s: None
2023-12-13 02:06:30,424 - INFO - beat: Starting...
2023-12-13 02:06:30,444 - INFO - Scheduler: Sending due task get_position (get_position)
2023-12-13 02:06:30,451 - INFO - Task get_position[05a34d95-0c63-4b76-b605-36677d58a177] received
2023-12-13 02:06:34,959 - INFO - Task get_position[05a34d95-0c63-4b76-b605-36677d58a177] succeeded in 4.506628980000002s: None
2023-12-13 02:07:30,442 - INFO - Scheduler: Sending due task get_position (get_position)
2023-12-13 02:07:30,445 - INFO - Task get_position[2f5e8caf-e950-4f5c-b131-329be2ca00b1] received
2023-12-13 02:07:35,317 - INFO - Task get_position[2f5e8caf-e950-4f5c-b131-329be2ca00b1] succeeded in 4.869326347999987s: None
